src/domain/schemas/document-schemas.js

```diff
@@ -1,8 +1,7 @@
 import joi from 'joi';
+import { idOrKeySchema } from './shared-schemas.js';
```

Comment:
Funny, I was introducing `generic-schemas` also in my current ticket 🧠 
Will use yours 👍

Reply:
Perfect